eng/tools/tasks/Microsoft.DotNet.UnifiedBuild.Tasks/WritePackageVersionsProps.cs

```diff
@@ -89,6 +90,8 @@ public class WritePackageVersionsProps : Microsoft.Build.Utilities.Task
         /// </summary>
         public string VersionDetails { get; set; }
 
+        public bool IsPreviousPropsFile { get; set; }
```

Comment:
Nit: I would change this to an ITaskItem[] PropertySuffixes with default being VersionPropertySuffix and VersionPropertyAlternateSuffix if not set. Then the caller would set this to 'PreviousVersion' when constructing the previously SB props.